Userland/Applications/TextEditor/TextEditorWidget.cpp

```diff
@@ -685,9 +686,27 @@ void TextEditorWidget::update_html_preview()
     m_page_view->scroll_into_view(current_scroll_pos, true, true);
 }
 
-void TextEditorWidget::update_statusbar_cursor_position()
+void TextEditorWidget::update_statusbar()
 {
     StringBuilder builder;
     builder.appendff("Line: {}, Column: {}", m_editor->cursor().line() + 1, m_editor->cursor().column());
+
+    if (m_editor->has_selection()) {
+        int word_count = 0;
+        bool in_word = false;
+        String selected_text = m_editor->selected_text();
+        for (char c : selected_text) {
+            if (in_word && isspace(c)) {
+                in_word = false;
+                word_count++;
+                continue;
+            }
+            if (!in_word && !isspace(c))
+                in_word = true;
+        }
+        if (in_word)
+            word_count++;
+        builder.appendff("        Selected: {} {} ({} {})", m_editor->selected_text().length(), m_editor->selected_text().length() == 1 ? "character" : "characters", word_count, word_count != 1 ? "words" : "word");
```

Comment:
We should reuse the `selected_text` variable we already created above instead of calling `selected_text()` again twice :)

Reply:
Oops how could I not notice that. Fixed :)